components/actiongroup/index.css

```diff
@@ -57,6 +57,12 @@
 	gap: var(--mod-actiongroup-vertical-spacing-regular, var(--spectrum-actiongroup-vertical-spacing-regular));
 	display: inline-flex;
 	flex-direction: column;
+
+	&.spectrum-ActionGroup--justified .spectrum-ActionGroup-item {
+		/* flex: unset removes the flex: 1 set on `.spectrum-ActionGroup--justified .spectrum-ActionGroup-item`,
+		in order to keep the action buttons from shrinking */
+		flex: unset;
+	}
```

Comment:
@5t3ph I know you suggested that maybe we should use `flex-grow: 1` as opposed to the `flex: 1` (which would also set flex-shrink which was probably causing our squish-ness), however, I was struggling to not undo the horizontal justified behavior.

I tried `flex-grow: 1` in a couple of places:
```
.spectrum-ActionGroup--justified .spectrum-ActionGroup-item {
	flex-grow: 1;
}

// OR
.spectrum-ActionGroup--vertical {
	gap: var(--mod-actiongroup-vertical-spacing-regular, var(--spectrum-actiongroup-vertical-spacing-regular));
	display: inline-flex;
	flex-direction: column;

	&.spectrum-ActionGroup--justified .spectrum-ActionGroup-item {
		flex-grow: 1;
	}
}
```
The first selector fixed the vertical justified button heights (from 36px to the correct 32px), but then unjustified the horizontal justified action group:
<img width="338" alt="Screenshot 2024-10-18 at 3 11 53 PM" src="https://github.com/user-attachments/assets/26e2b6be-0ed4-4766-a0b8-6a14c4c04b2e">

The second selector....didn't do anything? The action buttons were still the incorrect 36px, (and the icon-only aren't tall enough). 
<img width="414" alt="Screenshot 2024-10-18 at 3 28 58 PM" src="https://github.com/user-attachments/assets/13e8a045-f30f-4c2b-8ba0-2874c6126ca5">

With the new selector here with `flex: unset`, the horizontal justified remains the same, AND the vertical justified now is not squished and the proper 32px.

<img width="411" alt="Screenshot 2024-10-18 at 3 29 57 PM" src="https://github.com/user-attachments/assets/42b8d3fc-ddc9-4f00-8f6c-8148ff0da44a">

Am I missing anything with `flex-grow`? I know you mentioned `min-block-size` might be redundant, but would you rather that than the `flex: unset`? I haven't tried anything with that yet.

Reply:
After our pairing, we realized I just didn't understand what _should_ be happening in a justified actiongroup! `flex-grow: 1` is now up: https://github.com/adobe/spectrum-css/pull/3256/commits/85656a0cf06f3f61bbd052cb35fbfa1b378c9701